roles/glance/tasks/main.yml

```diff
@@ -49,23 +55,21 @@
     - glance-api
     - glance-registry
 
-- name: start glance services
-  service: name={{ item }} state=started
-  with_items:
-    - glance-api
-    - glance-registry
-
 - name: glance config
   template: src={{ item }} dest=/etc/glance mode=0644
   with_fileglob: ../templates/etc/glance/*
   notify:
     - restart glance services
 
-- name: permit access to glance
-  ufw: rule=allow to_port={{ item }} proto=tcp
+- name: sync glance database
+  command: glance-manage db_sync
+  when: database_create.changed or force_sync|default('false')|bool
+
+- name: start glance services
+  service: name={{ item }} state=started
   with_items:
-    - 9292
-    - 9293
+    - glance-api
+    - glance-registry
```

Comment:
This is still happening after the service restart. Service restart is happening at https://github.com/blueboxgroup/ursula/pull/670/files#diff-d69e0894556bd648ef3ab100be8f6569L29   line 29 due to a flush_handlers call, which is going to be pretty hard to sort out because the git clone of glance triggers both the pip install /and/ the glance restart. Perhaps taking the glance restart trigger out? We should probably chat and pair on this.